packages/cli/package.json

```
@@ -41,7 +41,7 @@
     "@oclif/command": "1.8.0",
     "@oclif/config": "1.17.0",
     "@oclif/plugin-autocomplete": "0.3.0",
-    "@oclif/plugin-help": "3.2.2",
+    "@oclif/plugin-help": "~3.2.14",
```

For this repo, we always pin the exact version of a dependency, to be extra safe that nothing breaks due to accidental version bump.

```suggestion
    "@oclif/plugin-help": "3.2.14",
```